_data/users.yml

```diff
@@ -122,6 +122,20 @@
   links:
     - urltext: View the Bazaar Technologies website
       url: https://www.bazaar-tech.com/
+- name: Bloomberg
+  highlight: false
+  anchor: bloomberg
+  logo: /assets/images/logos/bloomberg.png
+  logosmall: /assets/images/logos/bloomberg-small.png
+  testimonial: |
+    Trino is a critical piece of the data platform infrastructure at Bloomberg.
```

Comment:
That's it? That's like saying food is a critical part of nutrition.

Reply:
Hahah .. of course thats not it.. just a placeholder while we work on the real thing.